node-fs-readfile/cat.ts

```diff
@@ -0,0 +1,21 @@
+import * as fs from 'node:fs/promises';
+
+async function read(): Promise<void> {
+  try {
+    if (process.argv.length < 3) {
+      console.log('Invalid arguments.');
+      process.exit(1);
+    }
+
+    let result: string = '';
+    for (let i = 2; i < process.argv.length; i++) {
+      result =
+        result + (await fs.readFile(process.argv[i], { encoding: 'utf8' }));
```

Comment:
This works, but it runs all the promises in serial (one after the other). It's more efficient to use Promise.all, which runs them all in parallel (at the same time).